packages/sdk/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "casper-client-sdk",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

```suggestion
  "version": "1.0.2",
```
I misunderstood that the package was ready to release and bumped to 1.0.1 before finding the same grpc issue.  Need to bump to 1.0.2.

---

Thank you.